New ebuild for Froxlor 2.0.15.

SkyBeam opened the pull request:
It's very quickly done by updating soem file paths. I did not validate all the code paths and it's not very well tested yet. At least it worked fine on one of my personal systems to upgrade an Apache/PHP/MySQL/Postfix/Dovecot based system from 0.10.34 to 2.0.15.

d00p commented on www-apps/froxlor/froxlor-2.0.15.ebuild:
```diff
@@ -0,0 +1,278 @@
+	virtual/mysql
+	>=mail-mta/postfix-2.4[mysql,ssl=]
+	virtual/cron
+	>=dev-lang/php-7.1:*[bcmath,cli,ctype,curl,filter,gd,json(+),mysql,nls,pcntl,pdo,posix,session,simplexml,ssl=,tokenizer,unicode,xml,xslt,zlib,zip]
```
7.4 is minimum ;)

SkyBeam replied:
Changed to 7.4.

d00p commented on www-apps/froxlor/froxlor-2.0.15.ebuild:
```diff
@@ -0,0 +1,278 @@
+
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="awstats bind +dovecot fcgid fpm ftpquota lighttpd +log mailquota nginx pdns pureftpd quota ssl"
```
new traffic-tool "goaccess" as alternative to awstats/webalizer was introduced

SkyBeam replied:
Quickly added a goaccess use flag. No integration or pre-configuration of SQL done yet.

d00p reviewed (changes requested):
had not yet the chance to add all necessary changes fpr 2.0, the gentoo user group of froxlor seems to be rather small.
The whole "preparing/adjusting" the froxlor.sql.php needs to be re-checked

SkyBeam commented:
Well googling for gentoo froxlor overlay yields a [thread ](https://forum.froxlor.org/index.php?/topic/19383-gentoo-overlay-does-not-work-anymore/) where in Januaray 2020 the question about outdated repo was asked and my last request for an update in April 2020 ended in harsh response not to ask again as it's being worked on. So here we are 3 years later.

The question perhaps rather is: Is nobody using Frroxlor on Gentoo because there is no repo or is there no repo because nobody is using it. At least there seems to be some demand for it.

As there seems to be more active development I decided to quickly create an updated ebuild. It's far from being perfect as I am also not into Froxlor development and do not know the internals. Hence I would prefer an official ebuild of course. But my overlay at least allowed me to upgrade to latest version.
Kudos that the migration procedure built-in to Froxlor worked just fine.

d00p commented:
2.0.15 and 2.0.19 are not needed anylonger if you provide 2.0.21 - also the tar.gz checksums are missing in the manifest file.
Thanks for your effort on this - i'm really low on time and not a gentoo user anylonger for years now, your help is appreciated

SkyBeam commented:
Added checksums to Manifest.
Removed old builds.

I am also pretty new to this and the build is far from being perfect - but it worked for my installations and they might be usable for somebody. I am also short on time to invest so this is jsut quick and dirty attempt to share what I cobbled together based on your work. Thank you!
